css/ARN-framework.css

```diff
@@ -0,0 +1,524 @@
+.grid3 { background-color: lightcoral; }
+.grid4 { background-color: lightgreen; }
+.grid5 { background-color: lightskyblue; }
+.grid6 { background-color: lightpink; }
+.grid7 { background-color: lightsalmon; }
+.grid8 { background-color: lightyellow; }
+.grid9 { background-color: lightsteelblue; }
+.grid10 { background-color: lightslategray; }
+.grid11 { background-color: lightseagreen; }
+.grid12 { background-color: burlywood; }
+
+/* ANIMATIONS */
+.animate-grow { -webkit-animation: grow 0.5s ease 0s infinite alternate; animation: grow 0.5s ease 0s infinite alternate; }
+.animate-grow-hover { -webkit-animation: shrink 0.5s ease 0s 1 alternate; animation: shrink 0.5s ease 0s 1 alternate; animation-fill-mode: forwards; -webkit-animation-fill-mode: forwards; }
+.animate-grow-hover:hover { -webkit-animation: grow-h 0.5s ease 0s 1 alternate; animation: grow-h 0.5s ease 0s 1 alternate; animation-fill-mode: forwards; -webkit-animation-fill-mode: forwards; }
+.animate-slideUp { -webkit-animation: slideUp 0.5s ease 0s 1 alternate; animation: slideUp 0.5s ease 0s 1 alternate; }
+.animate-slideLeft { -webkit-animation: slideLeft 0.5s ease 0s 1 alternate; animation: slideLeft 0.5s ease 0s 1 alternate; }
+.animate-slideLeft-hover:hover { -webkit-animation: slideLeft-h 0.5s ease 0s 1 alternate; animation: slideLeft-h 0.5s ease 0s 1 alternate; }
+
+/* ANIMATION KEYFRAMES */
+@-webkit-keyframes grow {
+  from { -webkit-transform: scale(1); }
+  to { -webkit-transform: scale(1.05); }
+}
+
+@keyframes grow {
+  from { transform: scale(1); }
+  to { transform: scale(1.05); }
+}
+
+@-webkit-keyframes grow-h {
+  from { -webkit-transform: scale(1); }
+  to { -webkit-transform: scale(1.05); }
+}
+
+@keyframes grow-h {
+  from { transform: scale(1); }
+  to { transform: scale(1.05); }
+}
+
+@-webkit-keyframes shrink {
+  from { -webkit-transform: scale(1.05); }
+  to { -webkit-transform: scale(1); }
+}
+
+@keyframes shrink {
+  from { transform: scale(1.05); }
+  to { transform: scale(1); }
+}
+
+@-webkit-keyframes slideUp {
+  from { -webkit-transform: translateY(100px); }
+  to { -webkit-transform: translateY(0); }
+}
+
+@keyframes slideUp {
+  from { transform: translateY(100px); }
+  to { -webkit-transform: translateY(0); }
+}
+
+@-webkit-keyframes slideLeft {
+  from { -webkit-transform: translateX(100px); }
+  to { -webkit-transform: translateX(0); }
+}
+
+@keyframes slideLeft {
+  from { transform: translateX(100px); }
+  to { -webkit-transform: translateX(0); }
+}
+
+@-webkit-keyframes slideLeft-h {
+  from { -webkit-transform: translateX(100px); }
+  to { -webkit-transform: translateX(0); }
+}
+
+@keyframes slideLeft-h {
+  from { transform: translateX(100px); }
+  to { -webkit-transform: translateX(0); }
+}
+
+/* Media queries */
+
+/* Small devices (portrait tablets and large phones, 720px and up) */
+@media screen and (max-width: 720px), (min-width: 720px) {
+  /*    .bg-responsive{background-color: paleturquoise;} */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.